tests/AtlasConnectivity.Tests/AtlasConnectivity.Tests.csproj

```diff
@@ -2,14 +2,25 @@
 
   <PropertyGroup>
     <TargetFrameworks>net452;netcoreapp2.1</TargetFrameworks>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <LangVersion>6</LangVersion>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <WarningsAsErrors />
     <IsPackable>false</IsPackable>
     <CodeAnalysisRuleSet>..\..\MongoDBTest.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <AssemblyTitle>AtlasConnectivityTests</AssemblyTitle>
```

Comment:
Should Title and Product have `dot` separator?

Reply:
Yes. I intended these two values to match the assembly name, but I missed the dot.